nav_msgs/Path desired_path
float32 time_step
---
nav_msgs/Path smoothed_path
